auctioneerrpc/hashmail.proto

```diff
@@ -0,0 +1,118 @@
+syntax = "proto3";
+
+// We can't rename this to auctioneerrpc, otherwise it would be a breaking
+// change since the package name is also contained in the HTTP URIs and old
+// clients would call the wrong endpoints. Luckily with the go_package option we
+// can have different golang and RPC package names.
+package poolrpc;
+
+option go_package = "github.com/lightninglabs/pool/auctioneerrpc";
+
+// HashMail exposes a simple synchronous network stream that can be used for
+// various types of synchronization and coordination. The service allows
+// authenticated users to create a simplex stream call a cipher box. Once the
+// stream is created, any user that knows of the stream ID can read/write from
+// the stream, but only a single user can be on either side at a time.
+service HashMail {
```

Comment:
Reminder to add all those RPCs to the auth whitelist in aperture, otherwise a recipient with no channels won't be able to pay for an LSAT.